README.md

```diff
@@ -45,6 +45,12 @@ listener.on('close', function() {
 listener.close();
 </script>
 ```
+Testing
+-------
+1. Add pulse credentials to test profile in config.yml
```

Comment:
These should be in `user-config.yml` instead of `config.yml`. I see you added it to the `.gitignore` so I think you meant to say it here as well.

Whenever there's a `user-config.yml` to be used, I like to have a `user-config-example.yml` checked into the project.